docs/users/included-tools.md

```diff
@@ -0,0 +1,19 @@
+<h1>Developer Tools Included in the Container</h1>
+We have included several useful tools for Developers in our containers.
```

Comment:
s/included several useful tools for Developers/included several developer tools/ (definitely no capitalization for developers)